robocup_ws/src/Planner/src/team_0_master.py

```diff
@@ -3,19 +3,30 @@
 from team_master import TeamMasterServer, TeamMaster
 from goalkeeper_controller import Team1GoalkeeperController
 from stay_in_place_controller import NullController
+from player_controller import PlayerController
+from striker_controller import Team1StrikerController
 
 
 class TeamMaster0(TeamMaster):
     def __init__(self):
         team_id = 0
         super().__init__(team_id)
         self.goalkeeper_logic = Team1GoalkeeperController(0)
-        self.striker_left_logic = NullController()
-        self.striker_right_logic = NullController()
+        self.striker_left_logic = PlayerController()
+        self.striker_right_logic = Team1StrikerController()
         self.defence_left_logic = NullController()
         self.defence_right_logic = NullController()
         self.players_logic_was_updated = True
 
+    def plan(self):
+        self.striker_left_logic.get_coordinates(self.team_position.players_positions_efcs[3],
```

Comment:
It would be good if you can use the plan method from the parent class. If you want to modify default controllers actions then you can do sth similar to TeamMaster1.plan
```
def plan(self):
        super(TeamMaster1, self).plan()
        self.get_local_actions()
```

https://github.com/szgula/UofG_Robotics_TDP/blob/a7fddf38f5438ca2089b1e7bdb62ef6a77113c7a/robocup_ws/src/Planner/src/team_1_master.py#L26